### ISSUE TITLE
panic: runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by nikitastupin** [x1sec/commit-stream#2]
Hi @x1sec,

First of all thanks for the tool, I like the idea! Yesterday I've left it run in screen with `commit-stream -t TOKEN | cut -d ',' -f 2 >> all.txt` but at some point it crashed with SIGSEGV:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x628a10]

goroutine 1 [running]:
github.com/x1sec/commit-stream/pkg.Run(0x7ffc0b0d7023, 0x28, 0xc420082100, 0xc420082120)
        /home/ubuntu/go/src/github.com/x1sec/commit-stream/pkg/gitevents.go:113 +0x410
github.com/x1sec/commit-stream/pkg.DoIngest(0x7ffc0b0d7023, 0x28, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x6f1458)
        /home/ubuntu/go/src/github.com/x1sec/commit-stream/pkg/ingest.go:44 +0xbe
main.main()
        /home/ubuntu/go/src/github.com/x1sec/commit-stream/commit-stream.go:92 +0x304
```

**Comment by x1sec** [comment]
Thanks for reporting the issue. To give me an idea of how to trigger this issue, approximately how many rows are there in `all.txt`? Was there any abnormal output just before the crash?

**Comment by nikitastupin** [comment]
Unfortunately I've restarted the script writing output to the same file just after I've reported this issue. So I don't have the exact information right now.

However right now it's 996735 rows. Also I've did some analysis of strings in `all.txt` and found this string:

```
Get https://api.github.com/events?page=2&per_page=300: dial tcp: i/o timeout
```

Since it occurs exactly one time in the file and there is no other suspicious strings I suppose that it may be related to the crash.
